layouts/page/cudf-polars.html

```diff
@@ -0,0 +1,441 @@
+    data: {
+      labels: ['20GB', '40GB', '60GB'],
+      datasets: [
+        {
+          label: '.collect()',
+          barPercentage: 0.6,
+          data: [35,67,136], 
+          backgroundColor: 'rgb(33, 150, 243, 0.8)'
+        },
+        {
+          label: '.collect(engine="gpu")',
+          barPercentage: 0.6,
+          data: [12,19,32],
+          backgroundColor: 'rgb(115, 6, 255, 0.8)'
+        }]
+    },
+    options: {
+      indexAxis: 'x',
+      grouped: true,
+      scales: {
+        y: {
+          type: 'linear',
+          beginAtZero: true,
+          grid: {
+            display: true,
+            borderWidth: 0
+          },
+          title: {
+            display: true,
+            text: 'Time (s)'
+          }
+        },
+        x: {
+          ticks: {
+            font: {
+              weight: 'bold'
+            }
+          },
+          grid: {
+            display: false,
+            drawTicks: false,
+            borderWidth: 3
+          },
+          title: {
+            display: true,
+            text: 'Dataset Size (GB)'
+          }
+        }
+      },
+      plugins: {
+        legend: {
+          display: true
+        },
+        title: {
+          display: true,
+          text: 'PDS-H benchmark 80GB query processing time across a range of dataset sizes'
+        }
+      },
+      layout: {
+        padding: {
+          left: 0,
+          right: 20,
+          top: 0,
+          bottom: 0
+        }
+      }
+    }
+  });
+</script>
+
+<script>
+  const ctx2 = document.getElementById('perfchartspeed');
+  new Chart(ctx2, {
+    type: 'bar',
+    data: {
+      labels: ['Q13', 'Q17', 'Q18', 'Q21'],
+      datasets: [
+        {
+          label: 'Polars (CPU)',
+          barPercentage: 0.6,
+          data: [7,12,17,41],
+          backgroundColor: 'rgb(33, 150, 243, 0.8)'
+        },
+        {
+          label: 'Polars (GPU)',
+          barPercentage: 0.6,
+          data: [1,2,1,3],
+          backgroundColor: 'rgb(115, 6, 255, 0.8)'
+        }]
+    },
+    options: {
+      indexAxis: 'x',
+      grouped: true,
+      scales: {
+        y: {
+          type: 'linear',
+          beginAtZero: true,
+          grid: {
+            display: true,
+            borderWidth: 0
+          },
+          title: {
+            display: true,
+            text: 'Time (s)'
+          }
+        },
+        x: {
+          ticks: {
+            font: {
+              weight: 'bold'
+            }
+          },
+          grid: {
+            display: false,
+            drawTicks: false,
+            borderWidth: 3
+          },
+          title: {
+            display: true,
+            text: 'Query Number'
+          }
+        }
+      },
+      plugins: {
+        legend: {
+          display: true
+        },
+        title: {
+          display: true,
+          text: 'PDS-H benchmark 80GB top performing compute heavy queries with complex groupby and join operations'
```

Comment:
Can we make this -

Top performing compute heavy queries with complex groupby and join operations (PDS-Benchmark; 80 GB dataset)